lms/views/dashboard/api/course.py

```diff
@@ -8,12 +12,49 @@
     APICourses,
     CourseMetrics,
 )
-from lms.models import RoleScope, RoleType
+from lms.models import Course, RoleScope, RoleType
 from lms.security import Permissions
 from lms.services.h_api import HAPI
 from lms.services.organization import OrganizationService
+from lms.validation._base import PyramidRequestSchema
 from lms.views.dashboard.base import get_request_course, get_request_organization
 
+PAGINATION_LIMIT = 100
+"""Maximum numbr of items to return in paginated endpoints"""
+
+
+class ListCoursesSchema(PyramidRequestSchema):
+    location = "query"
+
+    organization = fields.Str()
```

Comment:
TODO, actually use this to filter the results.

Just as an example here that this endpoint (and the equivalent ones in assignments, users) will need filtering on top of pagination.